grafana/provisioning/alerting/cpu_temperature_alert_rule.yaml

```diff
@@ -139,9 +139,8 @@ groups:
         annotations:
           description: >-
             CPU {{ $labels.cpu }} on node {{ $labels.node_id }} has a temperature of
-            {{ $values.average_cpu_temperature.Value }} C which is over the
-            threshold of {{ $values.average_cpu_temperature_max.Value }} C.
-            Please verify that your cooling systems are functional.
+            {{ $values.average_cpu_temperature.Value }} °C. This value is over the
+            threshold of {{ $values.average_cpu_temperature_max.Value }} °C.
+            Verify that the cooling systems on your node are functioning correctly.
```

Comment:
This is probably fine wording, but the broken cooling systems may be in their data center rather than specifically on the node.

Reply:
Good point. Corrected!